### ISSUE TITLE
Support optional command parameters

### ISSUE THREAD

**Issue by ikkentim** [adambajguz/Typin#205]
I'd like to have an optional parameter for a command.

In the usage message this might look like: `dotnet MyApp.dll mycommand <RequiredParameter> <(OptionalParameter)>`.

Would it be possible to add this?

**Comment by adambajguz** [comment]
Yes, you can write a custom help writer, but I will think about this concept maybe `<RequiredParameter> <(OptionalParameter)>` should be in help by default.

**Comment by vladimir-kovalyuk** [comment]
+1
